timemachine/fe/single_topology.py

```diff
@@ -1205,27 +1168,7 @@ def assert_chiral_consistency(
     assert len(dst_chiral_restr_idx_set.allowed_set.intersection(src_chiral_restr_idx_set.disallowed_set)) == 0
 
 
-def check_chiral_validity_impl(
-    atom_map: AtomMapMixin,
-    src_chiral_restr_idx_set: ChiralRestrIdxSet,
-    dst_chiral_restr_idx_set: ChiralRestrIdxSet,
-    src_bond_idxs: NDArray,
-    dst_bond_idxs: NDArray,
-):
-    """Assert that we can directly turn on the chiral volumes (after bonds) without staggering angles."""
-
-    chiral_centers_in_mol_a = chiral_utils.find_chiral_atoms(atom_map.mol_a)
-    chiral_centers_in_mol_b = chiral_utils.find_chiral_atoms(atom_map.mol_b)
-
-    src_chiral_centers_in_mol_c = [atom_map.a_to_c[x] for x in chiral_centers_in_mol_a]
-    dst_chiral_centers_in_mol_c = [atom_map.b_to_c[x] for x in chiral_centers_in_mol_b]
-
-    check_chiral_validity_src_dst(src_chiral_centers_in_mol_c, dst_chiral_restr_idx_set, src_bond_idxs)
-    check_chiral_validity_src_dst(dst_chiral_centers_in_mol_c, src_chiral_restr_idx_set, dst_bond_idxs)
-
-
-def assert_chiral_consistency_and_validity(
-    atom_map: AtomMapMixin,
+def assert_chiral_consistency(
```

Comment:
* remove references to the distinction between `consistency` and `validity` from docstring,
* possibly inline the definition of `_assert_chiral_consistency` here, rather than keeping both functions `assert_chiral_consistency` and `_assert_chiral_consistency` with slightly different signatures? (but if it makes something convenient somewhere, keeping both signatures is fine)

Reply:
ah yeah, good points. Willl clean-up.